### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by ContentBrewmaster**
Folder for new IA structure about the guidebook.

### CODE REVIEW

File: docs/about-this-guidebook/git-workflow.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+Resources:
+
+- [Fork a Repo](https://help.github.com/articles/fork-a-repo/)
+- [Working with Forks](https://help.github.com/articles/working-with-forks/)
+- [Getting Started with GitHub Desktop](https://help.github.com/en/desktop/getting-started-with-github-desktop)
+- [Contributing to Projects with GitHub Desktop](https://help.github.com/desktop/guides/contributing/)
+
+## Assignment of pull requests
+
+Your PR will automatically be assigned to a team for a review, based on which part of the handbook you are editing. That automatic review assignment is explained in [docs-governance.md](docs-governance.md). If you think that certain individuals on the team may have particular subject matter expertise or authority on the topic, you can also assign them directly via the "Reviewers" section of the right hand sidebar.
```
Comment (dmundra) [inline docs/about-this-guidebook/git-workflow.md]: @ContentBrewmaster I think the link for `docs-governance.md` should be `guidebook-governance.md`.

File: docs/about-this-guidebook/github-actions.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# GitHub actions
+
+GitHub Actions CI is a hosted, distributed continuous integration service used to build and test projects hosted at GitHub. Actions automatically detects when a commit has been made and pushed to a GitHub repository, and each time this happens, it will try to build the project and run tests.
+
+Every time a pull request is made to the handbook repo, or when a commit is merged, GitHub CI will run tests to ensure that commit complies with certain standards. For now, it is primarily concerned with our [markdown](markdown.md) syntax, and will "fail" if there are errors as specified in the .remarkrc.error file in the root directory of this repo.
```
Comment (dmundra) [inline docs/about-this-guidebook/github-actions.md]: @ContentBrewmaster I think the link for `markdown.md` should be `markdown-for-guidebook.md`.

**Review by dmundra — CHANGES_REQUESTED** [review]
I found two links that need updating.

File: docs/about-this-guidebook/github-actions.md (hunk 2 above)
Comment (ContentBrewmaster) [reply docs/about-this-guidebook/github-actions.md]: Fixed both links

File: docs/about-this-guidebook/git-workflow.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+Resources:
+
+- [Fork a Repo](https://help.github.com/articles/fork-a-repo/)
+- [Working with Forks](https://help.github.com/articles/working-with-forks/)
+- [Getting Started with GitHub Desktop](https://help.github.com/en/desktop/getting-started-with-github-desktop)
+- [Contributing to Projects with GitHub Desktop](https://help.github.com/desktop/guides/contributing/)
+
+## Assignment of pull requests
+
+Your PR will automatically be assigned to a team for a review, based on which part of the handbook you are editing. That automatic review assignment is explained in [guidebook-governance.md](docs-governance.md). If you think that certain individuals on the team may have particular subject matter expertise or authority on the topic, you can also assign them directly via the "Reviewers" section of the right hand sidebar.
```
Comment (dmundra) [inline docs/about-this-guidebook/git-workflow.md]: @ContentBrewmaster looks like you fixed the link text but not the link itself, the `(docs-governance.md)` also needs the same update.

Comment (ContentBrewmaster) [reply docs/about-this-guidebook/git-workflow.md]: Fixed.

**Review by dmundra — APPROVED** [review]

**Pull request merged**